rest-api/src/main/resources/paths/apps/v1_apps_self_participants_demographics.yml

```diff
@@ -0,0 +1,24 @@
+get:
+    operationId: getDemographicUsersAppLevel
+    summary: Fetches all app-level demographics for all users within an app
+    description: Fetches all app-level demographics for all users within an app (app-level, done by app admin)
+    tags:
+        - Apps
```

Comment:
"Apps" is usually for APIs that interact with apps directly, not APIs that act at the App level. We should either leave this on _For Admins or we can add a new tag Demographics.

Reply:
I think it would make sense to have a new tag Demographics so that the study-level and app-level APIs are grouped together; right now I think they are separated.